### ISSUE TITLE
Just have a "choose a national flag" menu

### ISSUE THREAD

**Issue by jidanni** [openstreetmap/iD#9508]
### Description

Here the user is trying hard to simply input that he has found a standard Canadian flag.

Before he gets to this point, there should be a simple menu where he can pick the flag out of a list of 220 etc. countries.

(Yes, this flag was on US soil.)

### Screenshots

![20230220T180105](https://user-images.githubusercontent.com/1230959/220074312-10f6b17e-d98d-4b18-8d77-53b7f570ce07.jpg)

OK, he finally tries "Upgrade the tags" and it works.
But it could be a lot simpler.

**Comment by jidanni** [comment]
OK, so one should
- pick national in Flag Type
- type two letter country code into Flag Country
- select "Upgrade the tags"

Phew... not very straightforward.

**Comment by 1ec5** [comment]
There is a preset for the Canadian flag. Clicking “Upgrade the tags” will use that preset, or you can go back and search for “Canada” instead of “flagpole”. It shows up with a 🇨🇦 icon so you know it’s the one you’re looking for.

#7123 tracks making the Flag Country field usable without having to memorize ISO country codes.

**Comment by jidanni** [comment]
Yes I'm saying the minute the user just types an fl... distinguishing flag from factory, he should start seeing some flag colors that leads into a selection menu where he can just pick out the flag he wants, without stumbling along the way.

About 99% of the image above is just confusing to the user and he hopes he doesn't see it at all. And it even has confusion of the words update and upgrade.

**Comment by 1ec5** [comment]
Perhaps the field proposed in #7123 could have flag emoji icons inside it, in case the user knows what the flag looks like but doesn’t know the country. Searching the presets would be a better experience though, since the existing icons are larger and easier to read.

Note that OSM also maps flags besides national flags. The Flag Wikidata field lets you search for a wider variety of flags. #7460 and #5566 would improve the field’s formatting so that the dropdown menu would also include icons of flags.

**Comment by jidanni** [comment]
OK.
I'm thinking sort of like when you have your cellphone virtual keyboard showing,
and at the bottom you see an emoji, and you tap it, and you see lots of emoji types,
one of which is a flag, and you tap it, and you see lots of  countries' flags, and you pick one.

Anyway, the challenge is: let's pretend we are talking about an illiterate editor from some underdeveloped country.
Well, the whole sequence needs to be picture based, right from the start.
Hmmm, I'll even make that into a new feature request!
Wait, it is already fully packed with icons... except after we type 'flagpole'.

**Comment by tyrasd** [comment]
let's handle this in #7123
